### PR TITLE
Added edges to buttons

### PR DESCRIPTION
**Pull request by nessemis**

### CODE REVIEW

File: MeesGame/UI/Interfaces/IMeasuredDimensions.cs [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 namespace MeesGame
 {
-    interface IMeasuredDimensions
+    internal interface IMeasuredDimensions
     {
         Vector2 MeasuredDimensions();
     }
-}
+}
```
Comment (jensj12) [inline MeesGame/UI/Interfaces/IMeasuredDimensions.cs]: Don't remove newlines at file end

**Comment by nessemis** [comment]
@jensj12  that one slipped through. Fixed now

**Comment by jensj12** [comment]
Please provide a description of what you've done. I expected more than just a plain border around every button. Make the border light up instead of using a key overlay when hovering over a button.

File: MeesGame/UI/DefaultUIValues.settings [hunk 2]
```diff
@@ -29,5 +26,14 @@
...
+      <Value Profile="(Default)">8</Value>
+    </Setting>
+    <Setting Name="DefaultButtonEdgeColor" Type="System.Drawing.Color" Scope="Application">
+      <Value Profile="(Default)">SlateGray</Value>
+    </Setting>
+    <Setting Name="DefaultButtonSelectedEdgeColor" Type="System.Drawing.Color" Scope="Application">
+      <Value Profile="(Default)">Gold</Value>
+    </Setting>
   </Settings>
 </SettingsFile>
```
Comment (LisanneW) [inline MeesGame/UI/DefaultUIValues.settings:39]: Missing newline

File: MeesGame/UI/Components/Texture.cs [hunk 3]
```diff
@@ -23,10 +23,20 @@ public override void DrawTask(GameTime gameTime, SpriteBatch spriteBatch, Vector
...
+                texture?.Dispose();
+                texture = value; }
+            get { return texture; }
+        }
+
         public Color Color
         {
             get { return color; }
-            set { color = value; }
+            set { color = value;
```
Comment (jensj12) [inline MeesGame/UI/Components/Texture.cs]: Format this as a normal function

File: MeesGame/UI/Components/Texture.cs [hunk 4]
```diff
@@ -23,10 +23,20 @@ public override void DrawTask(GameTime gameTime, SpriteBatch spriteBatch, Vector
             base.DrawTask(gameTime, spriteBatch, anchorPoint);
         }
 
+        public Texture2D CurrentTexture
+        {
+            set {
+                texture?.Dispose();
+                texture = value; }
+            get { return texture; }
```
Comment (jensj12) [inline MeesGame/UI/Components/Texture.cs:28]: Put the get before the set, and format set as a normal function

**Review by jensj12 — CHANGES_REQUESTED** [review]

**Review by LisanneW — CHANGES_REQUESTED** [review]
Invalidate the color picker when moving to a new tile.

**Comment by nessemis** [comment]
@LisanneW fixed the colorpicker not showing up/being removed.

**Review by jensj12 — APPROVED** [review]

**Review by LisanneW — APPROVED** [review]

**Pull request merged**